### PR TITLE
Logger defaults to consoleHandler when no handler is provided.

### PR DESCRIPTION
**Pull request by WouterSpaak**
This PR will make sure the logger uses `consoleHandler` by default when no `logHandler` is provided.

Fixes [4](https://github.com/politie/informant/issues/4).

### CODE REVIEW

File: src/logger.spec.ts [hunk 1]
```diff
@@ -382,6 +382,31 @@ describe('Logger', () => {
     });
 });
 
+describe('Logger', () => {
+    const called = stub();
+    let infoFn: any;
+    beforeEach(() => {
+        infoFn = console.info;
```
Comment (pavadeli) [inline src/logger.spec.ts]: It is better to use the Sinon API: `const consoleInfo = stub(console, 'info');` and later on: `consoleInfo.restore();`.

File: src/logger.ts [hunk 2]
```diff
@@ -161,6 +161,12 @@ function createLogMethod(level: LogLevel): LogMethod {
         }
         if (arguments.length) {
             const record = createLogRecord.apply(this, arguments);
+
+            // If no log handlers are registered, default to consoleHandler.
+            if (logHandlers.length === 0) {
+                registerLogHandler(consoleHandler);
```
Comment (pavadeli) [inline src/logger.ts]: I prefer not to register a handler at this point. If should be possible to register a handler after the first log statement without having to remove the consoleHandler. So simply *use* the `consoleHandler`, but do not register it.

File: package-lock.json [hunk 3]
```diff
@@ -46,7 +46,8 @@
         "abbrev": {
```
Comment (pavadeli) [inline package-lock.json]: Do not check-in `package-lock.json` when you have no changes to `package.json`.

**Comment by WouterSpaak** [comment]
@pavadeli Requested changes have been made.

**Pull request merged**